### ISSUE TITLE
[COGNITION]: GIBCT - Caution flag skip links SHOULD work for all screen readers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7986]
**Feedback framework**
- **⚠️ Should** fix this issue 1-3 sprints post launch
- [508 defect 3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Description

The skip to alert details link is created using an `<a>` tag with no anchor, just a `title` attribute. This works well for some screen readers, but fails for others. Recommend we change this to use a more robust approach, where the heading being targeted has an ID, and a tab index. Screenshot below.

WebAIM offers a good [article on skip links](https://webaim.org/techniques/skipnav/) that I've used in the past as my starting point. We have found in testing that IE11 needs a `tabIndex="-1"` on the target heading to behave correctly.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Skip link works for all screen readers (will be tested with JAWS, NVDA, VoiceOver, iOS VoiceOver)

## WCAG or Vendor Guidance (optional)

* [Bypass Blocks: Understanding SC 2.4.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-14 at 8 34 47 AM](https://user-images.githubusercontent.com/934879/79230952-fdccb900-7e2a-11ea-8b07-f1ac0f4454e5.png)

**Comment by zurbergram** [comment]
Requested changes appear to have been done in work for https://github.com/department-of-veterans-affairs/va.gov-team/issues/7867

**Comment by sonntag-adam** [comment]
![Screen Shot 2020-04-17 at 1.45.25 PM.png](https://images.zenhubusercontent.com/5c98df9c534e09000148f8d1/77488a76-be7d-4496-ab54-ae921a29bd71)

In staging @1Copenut

**Comment by 1Copenut** [comment]
@sonntag-adam This is working as expected in staging. Would you mind putting in a follow up ticket to remove the yellow halo from the targeted `<h3>` tag? Best practice is to put the halo on interactive elements only. Much appreciated.

/cc @desireeturner 

---
![Screen Shot 2020-04-20 at 10 01 27 AM](https://user-images.githubusercontent.com/934879/79772914-5651f980-82f6-11ea-8a2e-58ad433bbcc5.png)
